web/gui-v2/src/components/ListViewTable.jsx

```diff
@@ -239,6 +257,16 @@ columnDefinitions.forEach((colDef) => {
     } else {
       SLIDER_NATURAL_COLUMNS.push(colDef.key);
     }
+
+    if ( colDef?.isGrowthStat || colDef?.isPercent ) {
+      SLIDER_PERCENTAGE_COLUMNS.push(colDef.key);
+    }
+
+    if ( colDef?.aggregateType === 'sum' ) {
```

Comment:
I suggest we make this a default so we don't have to specify it in the majority of cases